tests/store/tracking/test_file_store.py

```diff
@@ -533,12 +534,15 @@ def test_create_experiment(self):
 
     def test_create_experiment_appends_to_artifact_uri_path_correctly(self):
         cases = [
-            ("path/to/local/folder", "path/to/local/folder/{e}"),
+            ("path/to/local/folder", "{cwd}/path/to/local/folder/{e}"),
             ("/path/to/local/folder", "/path/to/local/folder/{e}"),
-            ("#path/to/local/folder?", "#path/to/local/folder?/{e}"),
-            ("file:path/to/local/folder", "file:path/to/local/folder/{e}"),
+            ("#path/to/local/folder?", "{cwd}/#path/to/local/folder?/{e}"),
```

Comment:
@dbczumar, I know we discussed this offline, but I just want to make sure about this change......if a relative path looks like a fragment/query param (which doesn't make sense for a "relative" URI), with this PR change, it will be resolved as a literal path. IDK if this test makes sense here, but we shouldn't allow such a relative path with fragment/query param to be passed in like this for artifact locations.

Reply:
If the user supplies just a fragment or query string for their artifact root directory or artifact URI, we should interpret it as a local filesystem path. Your changes to the test expectations make sense :)